Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/DccWalletInfoCalculationManager.kt

```diff
@@ -34,32 +36,26 @@ class DccWalletInfoCalculationManager @Inject constructor(
                 person.dccWalletInfo == null ||
                 person.dccWalletInfo.validUntilInstant.isBefore(now)
             ) {
-                updateWalletInfoForPerson(person)
+                updateWalletInfoForPerson(person, admissionScenarioId)
             }
         }
+        Result.Success
     } catch (e: Exception) {
         Timber.e(e, "Failed to run calculation.")
+        Result.Failure(e)
     }
 
-    suspend fun triggerCalculationAfterCertificateChange() = try {
+    suspend fun triggerCalculationAfterCertificateChange(
+        admissionScenarioId: String
+    ): Result = try {
         initCalculation()
         personCertificatesProvider.personCertificates.first().forEach {
-            updateWalletInfoForPerson(it)
+            updateWalletInfoForPerson(it, admissionScenarioId)
         }
+        Result.Success
     } catch (e: Exception) {
         Timber.e(e, "Failed to run calculation.")
-    }
-
-    /**
-     * Trigger [DccWalletInfo] calculation for specific person
-     */
-    suspend fun triggerCalculationForPerson(personIdentifier: CertificatePersonIdentifier) {
```

Comment:
Why was this removed? Also why was the option to re-calculate the wallet for a single person in the Test Menu removed?

Reply:
Test menu for this option was removed too

Reply:
okay.